examples/misc/cursor_demo_sgskip.py

```diff
@@ -3,19 +3,20 @@
 Cursor Demo
 ===========
 
-This example shows how to use matplotlib to provide a data cursor.  It
-uses matplotlib to draw the cursor and may be a slow since this
-requires redrawing the figure with every mouse move.
+This example shows how to use Matplotlib to provide a data cursor.  It uses
+Matplotlib to draw the cursor and may be a slow since this requires redrawing
+the figure with every mouse move.
 
 Faster cursoring is possible using native GUI drawing, as in
```

Comment:
There's also `https://matplotlib.org/gallery/widgets/cursor.html` which seems like a better builtin version of this example. It should at least be linked.

Given this, do we still need this example here? If so, it's probably only for educational puposes and that should be mentioned.